client/my-sites/upgrades/map-domain/index.jsx

```diff
@@ -141,8 +144,13 @@ var MapDomain = React.createClass( {
 	}
 } );
 
-module.exports = connect( state => (
-	{
-		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
+export default connect(
+	( state, ownProps ) => {
+		const selectedSite = ownProps.sites.getSelectedSite();
```

Comment:
Just like my previous comment above, I'd suggest that we don't introduce another instance of `sitesList.getSelectedSite()`. Then we'll also no longer need to pass `ownProps` to `mapStateToProps`.